app/components/quiz/QuizQuestion.js

```diff
@@ -19,20 +19,24 @@ export default React.createClass({
     return {colors: randomColors, isSelectedFlags: isSelected};
   },
 
-  answerClicked: function(id, selected) {
+  answerProcessed: function(id, selected, text=null) {
```

Comment:
Oh, I meant like in the body of this function, inside the `if (text === null) {` block, just call a different function called like `answerProcessedText`
but on second look this function isn't long enough to warrant splitting up, so we're good 👍